src/worker.py

```
@@ -1301,4 +1337,145 @@ async def on_fetch(request, env):
+        " JOIN users u ON u.id = CASE WHEN pc.requester_id=? THEN pc.addressee_id ELSE pc.requester_id END"
+        " WHERE (pc.requester_id=? OR pc.addressee_id=?) AND pc.status='accepted'"
+        " ORDER BY pc.created_at DESC"
+    ).bind(user["id"], user["id"], user["id"], user["id"]).all()
+    peers = []
+    for r in rows.results or []:
+        peers.append({
+            "connection_id": r["id"],
+            "peer_id": r["peer_id"],
+            "peer_name": await decrypt_aes(r["peer_name_enc"] or "", enc),
+            "connected_at": r["created_at"],
+        })
```

Ruff (0.15.7)

[warning] Use a list comprehension to create a transformed list

(PERF401)